novawallet/Common/Services/ExtrinsicService/Substrate/ExtrinsicSenderResolution/Delegation/DelegationResolutionGraph.swift

```diff
@@ -0,0 +1,243 @@
+import Foundation
+import SubstrateSdk
+
+enum DelegationError: Error {
+    case noCompatibleProxyType
+    case invalidMultisigParameters
+    case noValidPath
+    case emptyPaths
+    case disjointPaths
+}
+
+struct DelegationKey: Hashable {
```

Comment:
Probably the error above and this struct should be defined inside DelegationResolution namespace